.gitpod.yml

```diff
@@ -0,0 +1,24 @@
+# List the start up tasks. Learn more https://www.gitpod.io/docs/config-start-tasks/
+tasks:
+  - name: npm
+    init: npm install
+    command: npm run start
+
+# List the ports to expose. Learn more https://www.gitpod.io/docs/config-ports/
+ports:
+  - port: 3000
+    onOpen: open-preview
+
+github:
+  prebuilds:
+    addBadge: true
+    addComment: false
```

Comment:
Doesn't this req @vinzvinci to add the app to our org?

Reply:
> Doesn't this req @vinzvinci to add the app to our org?

Nope, plus it's already added, AFAIK.

Reply:
Oh if it's added then good, otherwise for prebuild I think we have to add the app to our org